src/Apps/Collect/Routes/Collection/index.tsx

```diff
@@ -123,23 +122,17 @@ export const CollectionApp: React.FC<CollectionAppProps> = props => {
           counts={collection.artworksConnection?.counts as Counts}
         />
 
-        {/* HOTFIX FIXME: This rail was causing an error if included in SSR render
-              pass and so it was deferred to the client.
-
-              See: https://github.com/artsy/force/pull/6137
-          */}
-        {collection.linkedCollections.length === 0 &&
-          typeof window !== "undefined" && (
-            <LazyLoadComponent threshold={1000}>
```

Comment:
This didn't do anything to begin with since it's a fragment container. There was a waypoint usage in that component used for tracking, maybe that's what it originally was doing.